docker/packetbeat/packetbeat.yml

```diff
@@ -0,0 +1,53 @@
+packetbeat.interfaces.snaplen: 1514
+packetbeat.interfaces.type: af_packet
+packetbeat.interfaces.buffer_size_mb: 250
+packetbeat.protocols:
+- type: dns
+  ports: [53]
+  include_authorities: true
+  include_additionals: true
+- type: http
+  ports: [8080,80,8000,5000,3000,8200,9200,5601]
+  tags: ["http"]
+  send_headers: true
+  send_all_headers: true
+  split_cookie: true
+  send_request: true
+  send_response: true
+- type: mysql
+  ports: [3306]
+- type: icmp
+  enabled: true
+packetbeat.flows:
+  timeout: 30s
+  period: 10s
+packetbeat.procs:
+  enabled: true
+  system.hostfs: "/hostfs/proc"
+fields: { interface: "${INTERFACE:eth0}" }
+processors:
+  - add_docker_metadata:
+      host: "unix:///var/run/docker.sock"
+  - add_host_metadata:
+      #extremely verbose if enabled
+      netinfo.enabled: false
+#setup.template.overwrite: true
+xpack.monitoring.enabled: true
+queue.mem:
+  events: 20000
+output.elasticsearch:
+  hosts: '${ELASTICSEARCH_HOSTS:elasticsearch:9200}'
```

Comment:
can remove these `output.elasticsearch` settings since they're the default and there's no way to change from `compose.py` for now

Reply:
ok